src/DataTable/TablePaginationMinimal.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { useContext } from 'react';
 import DataTableContext from './DataTableContext';
 import Pagination from '../Pagination';
+import { ArrowBackIos, ArrowForwardIos } from '../../icons';
 
 function TablePaginationMinimal() {
```

Comment:
Can we add `propTypes` for this component?

Reply:
This component does not receive any props, so I don't think `propTypes` is needed here